src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,111 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.AbstractChromeTest;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+    public SoftAssertions softly = new SoftAssertions();
+
+    /** Assert Browser title. */
+    @Test(groups = {"HW2"})
+    public void exercise1() {
+
+        /* Assert Browser title. */
+        softly.assertThat(driver.getTitle()).as("Browser title is incorrect").isEqualTo("Home Page");
+
+        /* Perform login. */
+        new WebDriverWait(driver, 5)
+                .until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("li.uui-profile-menu"))).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        /* Assert Username is loggined.*/
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).as("Roman Iovlev is not logged")
+                .isEqualTo("ROMAN IOVLEV");
+
+        /* Assert that there are 4 items on the header section are displayed and they have proper texts. */
+        List<WebElement> items = driver.findElements(By.cssSelector(".uui-navigation.nav.navbar-nav.m-l8>li"));
```

Comment:
Please, try not to use "magic numbers" in your tests. We might have no idea what 4 really means. In other words, you could declare constants that have meaningful names.